tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_sort_contacts.py

```diff
@@ -24,12 +24,16 @@ def setUp(self):
         self.sorted_contacts_name_by_last = sorted(self._contacts_name_list, key=lambda name: name[1])
 
     def test_sort_contacts(self):
+        order_by_last_name_default = False
+
         contacts_app = Contacts(self.marionette)
         contacts_app.launch()
 
         # if "Order by last name" switch is on, turn off it
         contacts_settings = contacts_app.tap_settings()
-        if contacts_settings.order_by_last_name:
+        self.assertEqual(contacts_settings.order_by_last_name, order_by_last_name_default)
+        # This test starts from when order_by_last_name is false
+        if order_by_last_name_default:
```

Comment:
might be possible that default pref could change this after restart, so wouldn't hurt I think.